app/models/user.rb

```diff
@@ -25,4 +30,15 @@ def is_first_login?
   def assign_default_role
     self.add_role(:user) if self.roles.blank?
   end
+
+  def assign_avatar
+    avatars = Dir.entries("app/assets/images/avatars-monsters")
+    random_index = rand(avatars.count)
+    avatar_file_name = avatars[random_index]
+    self.avatar.attach(
+      io: File.open("app/assets/images/avatars-monsters/#{avatar_file_name}"),
+      filename: avatar_file_name,
+      content_type: "image/png"
```

Comment:
Could we replace the hard-coded `"image/png"` with plucking off the file extension from the filename and calling `Mime::Type.lookup_by_extension(file_extension)`?

Reply:
Done